backend/games/word_guesser/WordGuesser.py

```diff
@@ -0,0 +1,112 @@
+from random import randint
+from typing import Dict, List
+
+MAX_GUESSES = 6
+
+# initialize valid guesses and possible solutions
+SOLUTIONS: str
+VALID_WORDS: str
+
+solutions_file = open("backend/games/word_guesser/WordGuesser_Solutions.txt")
+guesses_file = open("backend/games/word_guesser/WordGuesser_ValidGuesses.txt")
+SOLUTIONS = solutions_file.read()
+SOLUTION_LIST = SOLUTIONS.splitlines()
+
+VALID_WORDS = guesses_file.read()
+VALID_WORDS_LIST = VALID_WORDS.splitlines()
```

Comment:
Might want to turn this into a set instead of a list for faster lookup. Same on the above line.

Reply:
id prefer to keep these as lists, as changing them to sets does not allow they to be easily indexed. Multiple "TypeError: 
'set' object is not subscriptable" errors arise